Services/PHPUnit/test/ilGlobalSuite.php

```diff
@@ -25,7 +25,67 @@ class ilGlobalSuite extends PHPUnit_Framework_TestSuite
 	 * @return	bool
 	 */
 	public function hasInstalledILIAS() {
-		return false;
+		$ilias_ini_path = __DIR__."/../../../ilias.ini.php";
+
+		if(!is_file($ilias_ini_path)) {
+			return false;
+		}
+
+		$ilias_ini = new ilIniFile($ilias_ini_path);
+		$ilias_ini->read();
+		$client_data_path = $ilias_ini->readVariable("server", "absolute_path")."/".$ilias_ini->readVariable("clients", "path");
+
+		if(!is_dir($client_data_path)) {
+			return false;
+		}
+
+		$client_data_path = null;
+		$dir = opendir($client_data_path);
+		while($file = readdir($dir)) {
+			if ($file != "." && $file != ".." && is_dir($client_data_path."/".$file)) {
+				$client_name = $file;
```

Comment:
Ich glaube hier müsste man eigentlich den Client verwenden, der auch in der phpunit-config angegeben ist.